tools/topology/sof-ehl-rt5660.m4

```diff
@@ -0,0 +1,193 @@
+#
+# Topology for ELKHARTLAKE with rt5660 codec
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`ssp.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Elkhartlake DSP configuration
+include(`platform/intel/ehl.m4')
+include(`platform/intel/dmic.m4')
+
+DEBUG_START
+
+#
+# Define the pipelines
+#
+# PCM0 ----> volume -----> SSP0 BE dailink 0
+# PCM1 <---- volume <----- DMIC48k (dmic48k, BE dailink 1)
+
+ifelse(HDMI, `1',
+`
+# PCM5 ----> volume -----> iDisp1 (HDMI/DP playback, BE link 5)
+# PCM6 ----> volume -----> iDisp2 (HDMI/DP playback, BE link 6)
+# PCM7 ----> volume -----> iDisp3 (HDMI/DP playback, BE link 7)
+# PCM8 ----> volume -----> iDisp3 (HDMI/DP playback, BE link 8)
+')
+
+dnl PIPELINE_PCM_ADD(pipeline,
+dnl     pipe id, pcm, max channels, format,
+dnl     frames, deadline, priority, core)
+
+# Low Latency playback pipeline 1 on PCM 0 using max 2 channels of s32le.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
+	1, 0, 2, s32le,
+	1000, 0, 0,
+	48000, 48000, 48000)
+
+# Low Latency capture pipeline 2 on PCM 0 using max 2 channels of s32le.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-volume-capture.m4,
+	2, 0, 2, s32le,
+	1000, 0, 0,
+	48000, 48000, 48000)
+
+# Passthrough capture pipeline 3 on PCM 1 using max 4 channels.
+# 1000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-passthrough-capture.m4,
```

Comment:
dont' we want a DMIC mute switch?

Reply:
@plbossart 
OK. I will pipe-volume-capture.m4